### DISCUSSION TITLE
Profile contributing date bug

### DISCUSSION THREAD

**Discussion by chizukicn** [community/community#75684] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I switch contributing organizations in my profile, the year may be +1

https://github.com/community/community/assets/16945858/deaeb6e5-be6a-4b88-ab96-ba4cf78cbc4d

**Comment by csengineer1990** [comment]
Hi, I tested the issue you mentioned - but I couldn't reproduce the same issue when switching contributing organizations in my profile. I thought it might be a browser-related issue or not. If there's anything specific you did or any additional details, let me know so I can dig deeper.

**Reply by chizukicn** [reply]
Thanks! This problem no longer exists.
My browser is `Microsoft Edge 119.0.2151.72 (arm64)`, and my operating system is `macOS 14.1.1`
